src/libs/network/endpointHandlers.ts

```
@@ -387,10 +366,24 @@ export default class ServerHandlers {
-                        error: e,
+                        error: e.toString(),
                     }
                 }
+                break
 
+            case "nativeBridge":
+                payload = tx.content.data
+                var nativeBridgeResult = await handleNativeBridgeTx(
+                    payload[1] as NativeBridgeOperationCompiled,
+                )
+                if (nativeBridgeResult === null) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)